src/main/java/gov/nist/oar/ds/controller/DownloadController.java

```diff
@@ -144,7 +145,7 @@ public void setDownloadService(DownloadService downloadService) {
    */
   @RequestMapping(value = "/{dsId}", method = RequestMethod.GET)
 
-  public ResponseEntity<byte[]> downloadZipFile(String id) throws Exception {
+  public ResponseEntity<byte[]> downloadZipFile(@RequestParam(value = "id", required = true) String id) throws Exception {
```

Comment:
Som -- what happens if an id parameter is not present?

Reply:
@RayPlante  As per my email, Deoyani's PR should take care of all the exception handling scenarios for zip and single file download.. Once Deoyani's PR is merged, we can close this PR without merging into integration.

Reply:
concur.